util/build-release.sh

```diff
@@ -50,6 +50,9 @@ cp "$PROJECT_DIR/SECURITY.md" "$target_dir_sudo/share/doc/sudo-rs/sudo/SECURITY.
 cp "$PROJECT_DIR/COPYRIGHT" "$target_dir_sudo/share/doc/sudo-rs/sudo/COPYRIGHT"
 cp "$PROJECT_DIR/LICENSE-APACHE" "$target_dir_sudo/share/doc/sudo-rs/sudo/LICENSE-APACHE"
 cp "$PROJECT_DIR/LICENSE-MIT" "$target_dir_sudo/share/doc/sudo-rs/sudo/LICENSE-MIT"
+# Create sudoedit symlinks
+ln -s "$target_dir_sudo/bin/sudo" "$target_dir_sudo/bin/sudoedit"
+ln -s "$target_dir_sudo/share/man/man8/sudo.8" "$target_dir_sudo/share/man/man8/sudoedit.8"
```

Comment:
Maybe commit the symlink into the repo? Also this is creating a symlink to an absolute path that doesn't match where the file will be installed. It would need to be


```suggestion
ln -s sudo.8 "$target_dir_sudo/share/man/man8/sudoedit.8"
```

Reply:
I would rather use the same approach for the binary as for the manpage, and not have a dangling symlink in the repo somewhere for the binary.